packages/@webex/plugin-cc/test/unit/spec/features/Agent.ts

```diff
@@ -85,4 +85,29 @@ describe('Agent', () => {
     expect(webexMock.logger.log).toHaveBeenCalledWith('LOGIN API SUCCESS');
     expect(response).toBe(loginResponse);
   });
-});
\ No newline at end of file
+
+  describe('#stationLogout', () => {
```

Comment:
We should have tests added for changes in the cc.ts file as well and the same goes for AgentService.ts. And we should cover the failure response test case also

Reply:
Added tests for `cc` and `AgentService`. Removed tests from `Agent` as it's one liner now.